app/views/api/v2/ansible_variables/show.json.rabl

```diff
@@ -10,7 +10,7 @@ attributes :id, :variable, :ansible_role, :ansible_role_id, :description, :overr
 node do |ansible_variable|
   {
     :override_values => partial(
-      'api/v2/override_values/index',
```

Comment:
@ezr-ondrej should we have kept this one in core, like we did for base and main?

Reply:
Maybe, this should get merged anyway tho, as using the override_values index was probably missing the `ansible_variable_id`

Reply:
That being said if ansible is only one failing, I'd rather force providers to choose what to render, as usually they want something extra, but it gets missing, just because "it works with what's in core"